trdg/string_generator.py

```diff
@@ -101,9 +111,6 @@ def create_strings_randomly(length, allow_variable, count, let, num, sym, lang):
     if lang == "cn":
```

Comment:
This would override any `min_seq_len` and `max_seq_len` when using `lang == "cn"`